src/libs/antares/study/parameters.cpp

```diff
@@ -650,8 +658,26 @@ static bool SGDIntLoadFamily_P(Parameters& d, const String& key, const String& v
         return false;
     }
 
+    /* Format added in Antares 8.1 */
+    if (key == "playlist_active_years")
+    {
+        std::list<int> in_playlist;
+        value.split(in_playlist, ",");
+        // Check indices
+        for (int y : in_playlist)
+            if ((uint)y >= d.nbYears || y < 0)
+                return false;
+        // All indices good, change d.yearsFilter
+        for (uint y = 0; y < d.nbYears; ++y)
+            d.yearsFilter[y] = false;
+        for (int y : in_playlist)
+            d.yearsFilter[y] = true;
+        return true;
```

Comment:
So if we want to activate all but deactivate a few years, we need to indicate all the activated years ?

Reply:
right